fe/fe-core/src/main/java/com/starrocks/statistic/ColumnStatsMeta.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.statistic;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**
+ * Meta of column-level statistics
+ */
+public class ColumnStatsMeta {
+
+    @SerializedName("columnName")
+    private String columnName;
```

Comment:
Use ColumnId?

Reply:
in theory we should use it, but the whole statistics system is using columnname, it would be huge work to refactor all of them to use columnid

Reply:
leave a todo here. we can do the refactor later